GS01SUMO/GS01SUMO_R00001_ingest.csv

```diff
@@ -1,80 +1,80 @@
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/CTDMO_sn37-12567/*.hex,GS01SUMO-RII11-02-CTDMOR020,recovered_inst,Available,
+#mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/ctdbp01/ctdbp01*.DAT,GS01SUMO-RII11-02-DOSTAD031,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/ctdbp02/ctdbp02*.DAT,GS01SUMO-RII11-02-DOSTAD032,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/ctdbp03/ctdbp03*.DAT,GS01SUMO-RII11-02-DOSTAD033,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/ctdbp01/ctdbp01*.DAT,GS01SUMO-RII11-02-FLORDG031,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/ctdbp02/ctdbp02*.DAT,GS01SUMO-RII11-02-FLORDG032,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/ctdbp03/ctdbp03*.DAT,GS01SUMO-RII11-02-FLORDG033,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PCO2W_snC0103/*.DAT,GS01SUMO-RII11-02-PCO2WC051,recovered_host,Not Expected,Data not expected
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PCO2W_snC0103/SAMI*.txt,GS01SUMO-RII11-02-PCO2WC051,recovered_inst,Available,
+#mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PCO2W_snC0104/*.DAT,GS01SUMO-RII11-02-PCO2WC052,recovered_host,Not Deployed,Not deployed
+#mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PCO2W_snC0104/SAMI*.txt,GS01SUMO-RII11-02-PCO2WC052,recovered_inst,Not Deployed,Not deployed
+#mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PCO2W_snC0105/*.DAT,GS01SUMO-RII11-02-PCO2WC053,recovered_host,Not Expected,Data not expected
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PCO2W_snC0105/SAMI*.txt,GS01SUMO-RII11-02-PCO2WC053,recovered_inst,Available,
+#mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/imm/phsen1/phsen1*.DAT,GS01SUMO-RII11-02-PHSENE041,recovered_host,Not Expected,Data not available

```

Comment:
I think this should be /omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl16/PHSEN_snP0137/